packages/ember-test-app/app/styles/app.css

```diff
@@ -1 +1,8 @@
 @import url("marketing-theme.css");
+
+/* The following tag ensures that the footer is always at the bottom of the page (no matter the size of the page content). */
+div[data-theme="marketing"] {
```

Comment:
Can we add something a little more specific than any `div` with the `data-theme` of `marketing`? Perhaps a class or id. Also, this doesn't seem to be working on my machine.
![image](https://github.com/user-attachments/assets/3ccf44fb-f285-4dab-a6ca-61b67bc03a17)

Reply:
Would it be possible to add this css as a part of the theme generation process? This insures that this css in not forgotten when we create a new add with nrg-ui